src/main/java/step3/Racing.java

```diff
@@ -0,0 +1,48 @@
+package step3;
+
+import java.util.Random;
+
+public class Racing {
+
+    private int carNumber;
+    private int repeatNumber;
+
+    public Racing(int carNumber, int repeatNumber) {
+        this.carNumber = carNumber;
+        this.repeatNumber = repeatNumber;
+    }
+
+    public void racing() {
+        Car[] cars = makeCar(this.carNumber);
+        for (int i = 0; i < this.repeatNumber; i++) {
+            play(cars);
+            ResultView resultView = new ResultView();
+            resultView.result(cars);
+        }
+    }
+
+    public Car[] makeCar(int carNumber) {
+        Car[] cars = new Car[carNumber];
+        for (int i = 0; i < carNumber; i++) {
+            cars[i] = new Car(1);
```

Comment:
Car생성시에 초기 위치는 0으로 해주세요. 🙂 
```suggestion
            cars[i] = new Car(0);
```